torch/ao/quantization/fx/prepare.py

```diff
@@ -116,22 +117,43 @@
 # list of dtypes to not add observers to
 _DO_NOT_OBS_DTYPE_LIST = [int, float, torch.bool, None]
 
+_OBS_OR_FQ_TYPE = Union[ObserverBase, FakeQuantizeBase, Any]
```

Comment:
Why Any here?

Reply:
Any is for things like MiinMaxObserver.with_args(...), maybe we can improve this part as well in the new api